http://bugs.gentoo.org/207151

readline isnt actually used in this package, so drop the include

--- which-2.19/tilde/xmalloc.h
+++ which-2.19/tilde/xmalloc.h
@@ -23,12 +23,6 @@
 #if !defined (_XMALLOC_H_)
 #define _XMALLOC_H_
 
-#if defined (READLINE_LIBRARY)
-#  include "rlstdc.h"
-#else
-#  include <readline/rlstdc.h>
-#endif
-
 #ifndef PTR_T
 
 #ifdef __STDC__
